src/pages/Index.vue

```diff
@@ -2,18 +2,115 @@
+        <b-row>
+          <b-col lg="6" md="6" sm="12">
+        <h1>Mentorship...for you!</h1>
 
-    <p>Mentorship...for you!</p>
+        <p>An all in one program targeted to speed up your growth!</p>
+        <b-btn class=".btn">Apply Now</b-btn>
+            </b-col>
+            <b-col>
+              <img src="./assets/images/illustration.png" img-alt="two people sitting in front of a computer" width="400px">
+          </b-col>
+          </b-row>
+        </b-container>
+    </header>
+    <b-container class=".b-container">
+     <b-card-group deck>
+         <b-card class=".b-card"
+            title="Community"
+           img-src="./assets/images/gcc2019-grouppicture.jpg"
+           img-alt="Community members at Galaxy Community Conference, 2019"
+           img-top
+           border-radius="10px"
+           >
+            <b-card-text>
+               GMN is birthed from the Galaxy Community and has her at the center of affairs.
+             </b-card-text>
+          </b-card>
+          <b-card class=".b-card"
+              title="Collaboration"
+             img-src="./assets/images/linkedin-sales-solutions-QgYvORVDdd8-unsplash.jpg"
+             img-alt="Four hands joined together"
+             img-top
+             border-radius="10px"       
+              >
+             <b-card-text>
+                  The program is founded and sustained by the pillars of collaboration.
+              </b-card-text>
+           </b-card>
+           <b-card class=".b-card"
+                title="Growth"
+                img-src="./assets/images/krakenimages-Y5bvRlcCx8k-unsplash.jpg"
+                img-alt="Corporate lady smiling"
+                img-top
+                border-radius="10px"       
+                >
+                <b-card-text>
+                    Your growth and expectations are our focus, and your growth drives us.
+                 </b-card-text>
+           </b-card>
+     </b-card-group>
+    </b-container>
+     <b-container class=".b-container">
+       <b-row style="background-color:#e9ecef;" class="p-4">
+         <b-col lg="6" md="6" sm="12">
+           <img src="./assets/images/gcc2019-grouppicture.jpg" alt="image">
+         </b-col>
+         <b-col lg="6" md="6" sm="12">
+           <h3>About Galaxy Mentor Network Program</h3>
+           <p>Galaxy Mentor Network (GMN) program is an initiative by the Galaxy Community to build her members in open source projects.</p> 
+            <p>The GMN program is for everyone who needs guidance in an open source project, anyone who needs help learning how to contribute to open source projects, guidance in navigating and settling into the Galaxy ecosystem.</p>
+            <a href="#">Learn More</a><span class="fas fa-arrow-circle-right"></span>            
+         </b-col>
+       </b-row>
+     </b-container>
+     
+     <b-container class=".b-container">
+       <b-row class="p-4">
+         <b-col lg="6" md="6" sm="12">
```

Comment:
Same comment as before here about the column

Reply:
I'm currently working on another branch and effecting the corrections. When I removed `lg`, `md`, and `sm`, the columns were no longer responsive

Reply:
Interesting. It should still be responsive. I will have a look later.
Another question. Why making the changes in another branch and not this one?

Reply:
Okay....Thank you!

We switched to a different branch because this branch kept giving us an error when we ran `npm run develop`. We switched to a new branch and we were able to decode the issue and I continued working there, effecting these corrections

Reply:
I'm also scared of this failed check...if I copy the changes onto this branch, it may still carry those commits :(

Is it okay with you to keep working with the new branch?

Reply:
Yes ok for me the new branch